sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -42,6 +43,10 @@ const Lottery = (props: { listing: Listing }) => {
   const { listingsService, profile } = useContext(AuthContext)
   const { data } = useFlaggedApplicationsMeta(listing?.id)
   const duplicatesExist = data?.totalPendingCount > 0
+  const shouldExpireData =
+    !profile?.userRoles.isAdmin && listing?.status === ListingsStatusEnum.closed
```

Comment:
I think we can only access this page if the listing is closed, curious why this check is here?

Reply:
I was following logic from the application expiration. Was definitely carried over but don't need here. will remove!